src/buffer/out/AttrRow.cpp

```diff
@@ -11,10 +11,16 @@
 // - attr - the default text attribute
 // Return Value:
 // - constructed object
-// Note: will throw exception if unable to allocate memory for text attribute storage
-ATTR_ROW::ATTR_ROW(const UINT cchRowWidth, const TextAttribute attr)
+ATTR_ROW::ATTR_ROW(const UINT cchRowWidth, const TextAttribute attr) noexcept
 {
-    _list.push_back(TextAttributeRun(cchRowWidth, attr));
+    try
+    {
+        _list.push_back(TextAttributeRun(cchRowWidth, attr));

```

Comment:
while we're here, I imagine we can make this an `emplace_back` (i know the compiler likely elides the double copy, but why not?)

Reply:
Good call!  Changing this actually saved another ~5ms of startup time on my machine.